src/analytics.js

```diff
@@ -0,0 +1,100 @@
+define([], function () {
+    function _workflow(message) {
+        if (window.analytics) {
+            window.analytics.workflow(message);
+        }
+    }
+
+    function _usage(label, group, message) {
+        if (window.analytics) {
+            window.analytics.usage(label, group, message);
+        }
+    }
+
+    function _fbUsage(group, message) {
+        _usage("Form Builder", group, message);
+    }
+
+    function clickedSave() {
+        _workflow("Clicked Save in the form builder");
+    }
+
+    function clickedGoToQuestion() {
```

Comment:
I didn't look closely at how many times each of these functions is referenced, but it looks like once for most of them. I'd be happy with this module having three functions: `workflow`, `usage`, and `fbUsage`, which eliminate a bit of boilerplate, making most analytics calls a one or two liner instead of three or four.